automated-comments/scala/general/failed_parse.md

```diff
@@ -0,0 +1,3 @@
+There was a problem whilst parsing.
```

Comment:
```suggestion
There was a problem while parsing.
```

Reply:
I still think `whilst` is fine and preferred :P :P :P (even though I also changed it to `while` just because USA USA USA)